warmup/vm_instruments.py

```diff
@@ -66,12 +66,24 @@ def parse_instr_data(self):
             return None
         for p_exec in xrange(len(self.instr_data)):
             iterations = len(self.instr_data[p_exec])
-            cumulative_times = [self.instr_data[p_exec][i][1] for i in xrange(iterations)]
-            times_secs = [cumulative_times[0] / 1000.0]
-            for index in xrange(1, len(cumulative_times)):
-                times_secs.append((cumulative_times[index] - cumulative_times[index - 1]) / 1000.0)
-            assert len(times_secs) == len(cumulative_times)
-            self.chart_data.append([ChartData('GC', times_secs, 'GC events')])
+            jit_cumulative_times = [self.instr_data[p_exec][i][1] for i in xrange(iterations)]
+            gc_cumulative_times = list()
+            # Sum GC times over all collectors that ran in each iteration.
+            for i in xrange(iterations):
+                gc_iteration = 0
+                for collectors in self.instr_data[p_exec][i][2]:
+                    gc_iteration += collectors[-1]
+                gc_cumulative_times.append(gc_iteration)
+            jit_times_secs = [jit_cumulative_times[0] / 1000.0]
```

Comment:
Lines 79--82 (inclusive) turn the cumulative times into per-iteration times, they also convert milliseconds into seconds. Lines 77 and 78 convert the first datum into milliseconds, because it isn't handled by the loops on lines 79--82.